### PR TITLE
Add hosts

### PR DESCRIPTION
**Pull request by cityofships**

### CODE REVIEW

File: pornography-hosts [hunk 1]
```diff
@@ -1827,6 +1827,7 @@
 0.0.0.0 blueporns.com
 0.0.0.0 bluewebcams.com
 0.0.0.0 bn.premiumhdv.com
+0.0.0.0 bngpt.com
```
Comment (spirillen) [inline pornography-hosts:1830]: https://mypdns.org/my-privacy-dns/matrix/-/issues/3415

File: pornography-hosts [hunk 2]
```diff
@@ -7691,6 +7692,7 @@
 0.0.0.0 ns4w.org
 0.0.0.0 ns99.info
 0.0.0.0 nsfw.reddit.com
+0.0.0.0 nsfw.xxx
```
Comment (spirillen) [inline pornography-hosts:7695]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/1777

File: pornography-hosts [hunk 3]
```diff
@@ -8297,6 +8299,7 @@
 0.0.0.0 pimpwebcams.com
 0.0.0.0 pin-up-angels.net
 0.0.0.0 pinaccesscode.com
+0.0.0.0 pinayflix.tv
```
Comment (spirillen) [inline pornography-hosts:8302]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/1588

File: pornography-hosts [hunk 4]
```diff
@@ -8774,6 +8777,7 @@
 0.0.0.0 pornteens.xyz
 0.0.0.0 pornteentube.pro
 0.0.0.0 pornthor.com
+0.0.0.0 porntn.com
```
Comment (spirillen) [inline pornography-hosts:8780]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/110

File: pornography-hosts [hunk 5]
```diff
@@ -8886,6 +8889,7 @@
 0.0.0.0 pro-ana-angels.wetpaint.com
 0.0.0.0 profporn.co
 0.0.0.0 projectvoyeur.com
+0.0.0.0 promo-bc.com
```
Comment (spirillen) [inline pornography-hosts:8892]: https://mypdns.org/my-privacy-dns/matrix/-/issues/1087

File: pornography-hosts [hunk 6]
```diff
@@ -14000,13 +14004,13 @@
 0.0.0.0 www.pornroxxx.com
 0.0.0.0 www.pornstarslikeitbig.co.uk
 0.0.0.0 www.pornstereo.com
+0.0.0.0 www.porntn.com
```
Comment (spirillen) [inline pornography-hosts:14007]: https://mypdns.org/my-privacy-dns/porn-records/-/issues/110

**Review by spirillen — CHANGES_REQUESTED** [review]
You are clearly violating my license 

You are to add

1. the license
2. credit the source of work

https://mypdns.org/my-privacy-dns/porn-records/-/blob/master/LICENSE

**Comment by cityofships** [comment]
> You are clearly violating my license
> 
> You are to add
> 
> 1. the license
> 2. credit the source of work
> 
> https://mypdns.org/my-privacy-dns/porn-records/-/blob/master/LICENSE

?

@Sinfonietta

**Comment by spirillen** [comment]
> > You are clearly violating my license
> > You are to add
> > 
> > 1. the license
> > 2. credit the source of work
> > 
> > https://mypdns.org/my-privacy-dns/porn-records/-/blob/master/LICENSE
> 

This is not to Sinfonietta, this is to you @cityofships, these records seems more than likely to have been copied from my project as linked, and without a link and information about the LICENSE type you are in violation of the license and copyright.

Please fix it @cityofships you are free to use, redistribute etc. free of charge as long as the two issues are solved.

**Comment by Sinfonietta** [comment]
Sorry @cityofships I have blocked that Spirillen guy.  So annoying.

**Pull request merged**